src/components/Examples/ExamplesRenderer.tsx

```
@@ -0,0 +1,179 @@
+  example,
+  apiKey,
+  activeLanguage,
+  setActiveLanguage,
+  children,
+}: PropsWithChildren<ExamplesRendererProps>) => {
+  const { id, files, visibleFiles, layout, products } = example;
+
+  const rewrittenFiles = useMemo<ExampleFiles>(() => {
+    return Object.entries(files).reduce((acc, [languageKey, languageFiles]) => {
+      return {
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)